scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -173,7 +173,7 @@ private fun getLicenseFindings(result: JsonNode, parseExpressions: Boolean): Lis
             }
         ).map { (licenseExpression, replacements) ->
             val spdxLicenseExpression = replacements.fold(licenseExpression.expression) { expression, replacement ->
-                expression.replace(replacement.scanCodeLicenseKey, replacement.spdxExpression)
+                expression.replace("\\b${replacement.scanCodeLicenseKey}\\b", replacement.spdxExpression)
```

Comment:
This still calls the string-version (not the regex-version) of `replace`, so you probably need to wrap this in `Regex(...)`.

Maybe also add a test case / modify an existing test case to demonstrate the issue?